tests/unittest/_torch/sampler/test_logits_logprobs.py

```
@@ -348,6 +348,96 @@ def test_sampled_token_always_in_prompt_logprobs(logprobs_k: int, simple_llm: LL
+    # Simple format must be list[float]; dict format must remain list[dict].
+    assert all(isinstance(x, float) for x in simple_gen_logprobs), (
+        f"Expected list[float], got element types: {[type(x) for x in simple_gen_logprobs]}"
+    )
+    assert all(isinstance(x, dict) for x in dict_gen_logprobs)
+
+    assert len(simple_gen_logprobs) == len(dict_gen_logprobs)
+    for token_id, lp_simple, lp_dict in zip(
+        dict_out.outputs[0].token_ids,
+        simple_gen_logprobs,
+        dict_gen_logprobs,
+    ):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)